### PR TITLE
List component created

### PR DESCRIPTION
**Pull request by aamrapaliwandhre**
**1)normal list
2)stacked list**
chrome-capture (18)

### CODE REVIEW

File: Components/list/list.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
+.list-container{
+    margin: 2rem 2rem 2rem 2rem;
```
Comment (harshkotkar5557) [inline Components/list/list.css]: you can use only `margin:2rem;` instead of repeating it

File: Components/list/list.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+.group-list{
+    width: 40rem;
+    background-color: var(--white);
+}
+.list-item{
+    font-size: 1.5em;
+    border: 1px solid black;
+    /* padding: ; */
```
Comment (harshkotkar5557) [inline Components/list/list.css]: you can remove commented part.

File: Components/list/list.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    font-size: 1em;
+    margin-left: 1rem;
+}
+
+.notification-avatar{
+    width: 4rem;
+    height: 4rem;
+    font-size: 0.8rem;
+    margin: 0.5rem 0.5rem 0.5rem 0.5rem;
+}
```
Comment (harshkotkar5557) [inline Components/list/list.css]: you can use only margin:0.5rem; instead of repeating it

File: Components/list/list.css [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+.ordered-list{
+    font-family: poppin;
+    font-size: 1.5em;
+}
+
+.unordered-list{
+    font-family: poppin;
+    font-size: 1.5em;
```
Comment (harshkotkar5557) [inline Components/list/list.css]: you are applying same properties to both classes so you can merge it using `ordered-list ,unordered-list{
font-family: poppin;
    font-size: 1.5em;
}
`

**Review by harshkotkar5557 — APPROVED** [review]
Looks clean, small suggestion though.

File: Components/list/list.css (hunk 1 above)
Comment (aamrapaliwandhre) [reply Components/list/list.css]: okay ...will do that..every time i do this same mistake : (

**Review by Mahi24-prog — APPROVED** [review]
Looks good 👍

**Pull request merged**